components/formats-gpl/src/loci/formats/in/NativeND2Reader.java

```diff
@@ -2412,9 +2412,10 @@ private void parseText(String textString, int offsetCount, boolean useDimensions
 
       lines = textString.split(" ");
       for (int i=0; i<lines.length; i++) {
-        String key = lines[i++];
-        while (!key.endsWith(":") && key.indexOf('_') < 0 && i < lines.length) {
-          key += " " + lines[i++];
+        StringBuilder sb = new StringBuilder(lines[i++]);
+        while ((sb.length() == 0 || sb.charAt(sb.length() - 1) != ':') && sb.indexOf("_") < 0 && i < lines.length) {
```

Comment:
`sb.isEmpty()` may read more clearly

Reply:
```StringBuilder``` doesn't seem to have an ```isEmpty``` method, so leaving this as-is.

Reply:
Indeed not! (Sorry, recovering from a cold, clearly still more addled than I'd realized.)